### PR TITLE
Add snapshot endpoints

### PR DESCRIPTION
**Pull request by gnvk**

### CODE REVIEW

File: alpaca_trade_api/entity_v2.py [hunk 1]
```diff
@@ -102,5 +103,35 @@ def __init__(self, raw):
...
+
+class BarV2(Remapped, _NanoTimestamped, Entity):
+    _tskeys = ('t',)
+
+    def __init__(self, raw):
+        super().__init__(bar_mapping_v2, raw)
+
+
+class SnapshotV2:
+
```
Comment (kalmant) [inline alpaca_trade_api/entity_v2.py:118]: Did flake8 create these empty lines (in `SnapshotV2` and `SnapshotsV2`)

File: alpaca_trade_api/rest.py [hunk 2]
```diff
@@ -652,6 +653,19 @@ def get_latest_quote(self, symbol: str) -> QuoteV2:
...
         return self.response_wrapper(resp['quote'], QuoteV2)
 
+    def get_snapshot(self, symbol: str) -> SnapshotV2:
+        """Get the snapshot for the given symbol"""
+        resp = self.data_get('/stocks/{}/snapshot'.format(symbol),
+                             api_version='v2')
+        return self.response_wrapper(resp, SnapshotV2)
+
+    def get_snapshots(self, symbols: List[str]) -> SnapshotV2:
+        """Get the snapshot for the given symbol"""
```
Comment (kalmant) [inline alpaca_trade_api/rest.py:663]: Incorrect docstring

File: alpaca_trade_api/entity_v2.py (hunk 1 above)
Comment (gnvk) [reply alpaca_trade_api/entity_v2.py:118]: No, removed them

**Review by kalmant — APPROVED** [review]
LGTM

**Pull request merged**